overlay/files/system/oem/21_kcrypt.yaml

```diff
@@ -2,10 +2,11 @@ name: "Update discovery plugins"
 stages:
     after-upgrade:
     - name: "Update plugins"
+      if: "[ $(kairos-agent state get oem.found) == 'true' ]"
```

Comment:
this actually gates the block if there is no oem partition found (ARM, for instance)